src/Calculator.ts

```diff
@@ -72,20 +118,109 @@ export function calculate(
+    };
+    report.status = R4.MeasureReportStatusKind._complete; // are there cases where this should be _pending or _error?
+    report.type = R4.MeasureReportTypeKind._individual; // are there cases where this should be _subjectList or _summary or _dataCollection
+
+    // measure url from measure bundle
+    const measureEntry = measureBundle.entry?.find(e => e.resource?.resourceType === 'Measure');
+    const measure = measureEntry?.resource as R4.IMeasure;
+    report.measure = measure.url || 'UnknownMeasure'; // or some other default?
+
+    // create group population counts from result's detailedResults (yes/no->1/0)
+    report.group = [];
+    result?.detailedResults?.forEach(dr => {
+      // TODO: check the measure definition for stratification to determine whether to add group.stratiier
+      // if yes, add stratifier with population copied into. Set counts to 0 if the result for the stratifier is false
+      const group = <R4.IMeasureReport_Group>{};
+      const detail = dr;
+      group.id = detail.groupId;
+      group.population = [];
+      let numeratorCount = 0.0;
+      let denominatorCount = 0.0;
+      // TODO: handle EXM111 (doesn't identify itself as a episode of care measure). if it's an episode of care, you need to iterate over
+      // stratifications : may need to clone results for one population group and adjust (in this case, just a straight clone)
+      detail?.populationResults?.forEach(pr => {
+        const pop = <R4.IMeasureReport_Population>{};
+
+        pop.code = {
+          coding: [
+            {
+              system: 'http://terminology.hl7.org/CodeSystem/measure-population',
+              code: pr.populationType,
+              display: POPULATION_DISPLAY_MAP[pr.populationType]
+            }
+          ]
+        };
+        pop.count = pr.result ? 1 : 0;
+        if (pr.populationType == PopulationType.NUMER) numeratorCount += pop.count;
+        if (pr.populationType == PopulationType.DENOM) denominatorCount += pop.count;
+        group.population?.push(pop);
+      });
+      // TODO: handle ratio or continuous variable or cohort
+      // currently assumes patient-based proportion measure http://hl7.org/fhir/us/cqfmeasures/measure-conformance.html#proportion-measures
+      group.measureScore = {
+        value: (numeratorCount / denominatorCount) * 1.0
+      };
+      report.group?.push(group);
+    });
+
+    // TODO: create contained evalatuated resource (contains patient... and population and entity information?)
+    // may eventually need the other bits (all pieces of information that were used in calculation), but we might be able to ignore for now
+    const evalId = uuidv4();
+    const contained: R4.IBundle = {
+      resourceType: 'Bundle',
+      id: evalId,
+      type: R4.BundleTypeKind._collection,
+      entry: []
+    };
+    const patient = patientBundles[0].entry?.find(e => {
+      return e.resource?.resourceType === 'Patient';
+    })?.resource;
```

Comment:
Whoops, I think I missed this when reviewing @lmd59's PR, so I know this isn't actually a snippet you wrote. But shouldn't we not be accessing the first entry of patientBundles directly? We should have the ID of the patient in the detailed results and then do a find in the bundle based on that?

Reply:
We probably want to actually add all of the patients that get passed in to the contained resources list, right? So in theory we should just loop over `patientBundles` here and add them all (or loop over the list of patients coming back from the detailed results. Either way in theory this method can handle >1 patient, so we should handle that case here).

Reply:
This definitely needs to be replaced. We are inside a iteration over the list of `ExecutionResults`. Each of these results indicates the patientId so we should use that to find the bundle for this patient.  As of right now it would use the data from the first patient to build the MeasureReport, so this would produce bad reports for any patient except the first one. Should I address this in my PR now or should this be addressed in later work?

Reply:
I think this should be addressed here. Otherwise the measure report logic will only work for one patient, right?

Reply:
Addressed in c09ce66. Can now run cli with list of paths to patient bundles. Existing calls with just one work as well.